app/assets/stylesheets/modules/badge.css

```diff
@@ -0,0 +1,20 @@
+.badge {
+  display: inline;
+  background-color: #e1e1e1;
+  color: black;
+  border: 1px solid #c0c0c0;
+  border-radius: 5px;
+  padding: 5px 10px;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.badge--success {
+  background-color: #c4f2c7;
+  border: 1px solid #9ecea1;
```

Comment:
Nice! You're speedy George! Thanks for the quick fix.
Since we're not modifying the `border-width` and `border-style` properties, we can simply isolate the specific border style we're revising:

`border-color: #9ecea1;`

Reply:
Great catch!!